nevergrad/functions/mixsimulator/core.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Based on https://github.com/Foloso/MixSimulator/tree/nevergrad_experiment
+
+import numpy as np
+#from nevergrad.parametrization import parameter as p
+from ..base import ExperimentFunction
+from mixsimulator.MixSimulator import MixSimulator  #type: ignore
+
+
+class OptimizeMix(ExperimentFunction):
```

Comment:
> you mean an example of code?

No that's not needed, but that actually reminds me that a quick description here as a docstring will be really helpful for other users, so that they understand in a few lines what this test case is about.